packages/block-editor/src/components/color-style-selector/index.js

```diff
@@ -78,14 +79,22 @@ const renderToggleComponent = ( { TextColor, BackgroundColor } ) => ( {
 	);
 };
 
-const BlockColorsStyleSelector = ( { children, ...other } ) => (
-	<Dropdown
-		position="bottom right"
-		className="block-library-colors-selector"
-		contentClassName="block-library-colors-selector__popover"
-		renderToggle={ renderToggleComponent( other ) }
-		renderContent={ () => children }
-	/>
-);
+const BlockColorsStyleSelector = ( { children, ...other } ) => {
+	deprecated( `wp.blockEditor.BlockColorsStyleSelector`, {
+		alternative: 'block supports API',
+		since: '6.1',
+		version: '6.3',
+	} );
```

Comment:
Hey, is it time to remove this API or if judged too impactful, consider extending the version.

Reply:
I thought we were no longer removing deprecated components and APIs.

Reply:
I'm not sure we've came to a concrete decision yet. If you want to keep it, then update the version maybe.